stress/src/logs.rs

```diff
@@ -26,7 +27,7 @@ impl LogExporter for MockLogExporter {
     fn export(
         &self,
         _batch: LogBatch<'_>,
-    ) -> impl std::future::Future<Output = LogResult<()>> + Send {
+    ) -> impl std::future::Future<Output = OTelSdkResult> + Send {
```

Comment:
weird, this should have been caught in the build.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/2629 opened to track. yes something missing in CI!